tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/LoginSteps.cs

```diff
@@ -8,18 +11,25 @@ namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
     [Binding]
     internal sealed class LoginSteps
     {
+        private readonly Dictionary<string, string> HostUrls = new Dictionary<string, string>
+        {
+            {"identity server", "http://host.docker.internal:8070/"},
```

Comment:
It would be good to get the base addresses into configuration. This way we only need to configure it once. It can be a separate task if you see fit.